tests/func-tests/monitoring_test.go

```diff
@@ -172,6 +174,74 @@ var _ = Describe("[crit:high][vendor:cnv-qe@redhat.com][level:system]Monitoring"
 		}).WithTimeout(prometheousTimeout).WithPolling(prometheousPolling).WithContext(ctx).ShouldNot(BeNil())
 	})
 
+	Context("should fire the DeprecatedMachineType alert", Serial, func() {
+		const deprecatedMachineType string = "pc-q35-rhel7.6.0"
+
+		It("when a VM is using a deprecated machine type", func(ctx context.Context) {
+			By("Creating a VM with a deprecated machine type")
+			runStrategy := kubevirtcorev1.RunStrategyAlways
+			vm := &kubevirtcorev1.VirtualMachine{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "deprecated-vm",
+					Namespace: tests.TestNamespace,
+				},
+				Spec: kubevirtcorev1.VirtualMachineSpec{
+					RunStrategy: &runStrategy,
```

Comment:
Consider using `ptr.To` instead of defining the `runStrategy` variable; e.g.
```suggestion
			vm := &kubevirtcorev1.VirtualMachine{
				ObjectMeta: metav1.ObjectMeta{
					Name:      "deprecated-vm",
					Namespace: tests.TestNamespace,
				},
				Spec: kubevirtcorev1.VirtualMachineSpec{
					RunStrategy: ptr.To(kubevirtcorev1.RunStrategyAlways),
```